nx-dev/ui-customers/src/lib/enterprise-customers.tsx

```diff
@@ -38,388 +35,272 @@ import {
-            </a>
-            <a
-              href="https://www.paramountplus.com"
-              rel="noreferrer"
-              target="_blank"
-              className="flex items-center justify-center border-x border-b border-slate-200/20 p-12 transition hover:bg-slate-100/20 hover:text-slate-950 dark:border-slate-800/20 dark:hover:border-slate-600/20 dark:hover:bg-slate-600/10 dark:hover:text-white"
-            >
-              <ParamountIcon aria-hidden="true" className="h-14 w-14" />
-            </a>
-            <div className="col-span-2 flex items-center justify-center p-6 md:col-span-2">
-              <DownloadCaseStudy
-                title="Banking Case Study"
-                description="$7B European bank cuts CI times by 62% and boosts team productivity."
-                buttonHref="https://go.nx.dev/banking-case-study"
-              />
-            </div>
-            <a
-              href="https://www.fedex.com"
-              rel="noreferrer"
-              target="_blank"
-              className="flex items-center justify-center border-b border-l border-r border-slate-200/20 p-12 transition hover:bg-slate-100/20 hover:text-slate-950 dark:border-slate-800/20 dark:hover:border-slate-600/20 dark:hover:bg-slate-600/10 dark:hover:text-white"
-            >
-              <FedExIcon aria-hidden="true" className="h-28 w-28" />
-            </a>
-            <a
-              href="https://www.aa.com"
-              rel="noreferrer"
-              target="_blank"
-              className="flex items-center justify-center border-x border-b border-slate-200/20 p-12 transition hover:bg-slate-100/20 hover:text-slate-950 dark:border-slate-800/20 dark:hover:border-slate-600/20 dark:hover:bg-slate-600/10 dark:hover:text-white"
-            >
-              <AmericanAirlinesIcon aria-hidden="true" className="h-12 w-12" />
-            </a>
-            <a
-              href="https://zipari.com"
-              rel="noreferrer"
-              target="_blank"
-              className="flex items-center justify-center border-b border-r border-t border-slate-200/20 p-12 transition hover:bg-slate-100/20 hover:text-slate-950 dark:border-slate-800/20 dark:hover:border-slate-600/20 dark:hover:bg-slate-600/10 dark:hover:text-white"
-            >
-              <ZipariIcon aria-hidden="true" className="h-12 w-12" />
-            </a>
+      <div className="mt-8">
+        <div className="mx-auto max-w-7xl">
+          <CustomerIconGrid icons={firstCustomerIcons} />
+        </div>
+
+        <div className="col-span-2 border-y border-slate-200 bg-slate-50 py-24 sm:py-32 md:col-span-4 dark:border-slate-800 dark:bg-slate-900">
+          <div className="mx-auto">
+            <CustomerTestimonialCarousel />
+          </div>
+        </div>
+        <div className="mx-auto max-w-7xl">
+          <CustomerIconGrid icons={secondCustomerIcons} />
+
+          <div className="grid-cols grid justify-center gap-4 px-2 py-6 md:grid-cols-3">
```

Comment:
Please add some breathing room like `my-8` or `my-12`, and same for the cards below like `gap-8` or `gap-12`. 

In addition, now that we have 3 cards on the same row, the shadows are too pronounced, can you make them lighter please?

Reply:
done